packages/common/src/allo/backends/allo-v2.ts

```diff
@@ -555,6 +556,13 @@ export class AlloV2 implements Allo {
         return error(result);
       }
 
+      await this.waitUntilIndexerSynced({
```

Comment:
if you check a few lines above were already doing `await this.waitUntilIndexerSynced`, it can be removed ...